schema/query/fuzz.go

```diff
@@ -0,0 +1,51 @@
+// +build gofuzz
+
+package query
```

Comment:
I totally agree with that, for tests. This one is not a test but a build flag for go-fuzz. I'm not even sure it would work actually.